cmd/wego/gitops/cmd.go

```diff
@@ -75,11 +78,25 @@ func installRunCmd(cmd *cobra.Command, args []string) error {
 	fluxClient := flux.New(cliRunner)
 	kubeClient := kube.New(cliRunner)
 
+	var gitProviderToken string
+	gitProviderToken = os.Getenv("GITHUB_TOKEN")
+	if gitProviderToken == "" {
+		fmt.Print("Github token will be used to create deployment keys and pull requests: ")
+		input, err := term.ReadPassword(int(os.Stdin.Fd()))
+		if err != nil {
+			return errors.Wrap(err, "failed reading ssh key password")
+		}
+		gitProviderToken = string(input)
+	} else {
+		fmt.Println("Using env GITHUB_TOKEN")
```

Comment:
Sorry I should have caught this before. We need to be explicit about what we are doing. I suggest:

`Copying GITHUB_TOKEN from env to cluster secret for create deploy keys and pull requests`.

A little verbose maybe, but we need to be explicit about this.